src/bp/core/services/auth/auth-service.ts

```diff
@@ -44,11 +44,12 @@ export default class AuthService {
     return await this.findUser({ email }, selectFields)
   }
 
-  async checkUserAuth(email: string, password: string, newPassword?: string) {
+  async checkUserAuth(email: string, password: string, newPassword?: string, ipAddress: string = '') {
     const user = await this.findUserByEmail(email || '', ['password', 'salt', 'password_expired'])
 
     if (!user || !validateHash(password || '', user.password!, user.salt!)) {
       this.stats.track('auth', 'login', 'fail')
+      this.logger.info(`Login failed. User "${email}" from IP "${ipAddress}"`)
```

Comment:
Should we log this as info or debug? I don't know if this will pollute the logs, on the other hand there might not be a ton of admins that logs in. Your call!